### PR TITLE
view: fix user change password

### LINKED ISSUE #177: [BUG] Identify and access > Users > selected user > Change password does not verify the old password
**Describe the bug**
'Identify and access > Users > selected user > Change password' dialog has a 'Current password' field but it's not mandatory nor it's ever verified if that's really the current password.

**To Reproduce**
1. Go to 'Identify and access > Users > selected user'
2. Click on 'Change password'
3. Enter a new password in the 'Password' field and a wrong password in the 'Current password' field.
4. The password has been changed even though the current password was wrong.

**Expected behavior**
The password change should not be allowed without entering correct current password.

**Screenshots**
None

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro x64
 - Browser: Chrome
 - Version: 80.0.3987.116 (Official Build) (64-bit)

**Additional context**
None

### PR DESCRIPTION
**Pull request by shwstppr**
Fixes #177 

Added custom form with the conditional current password field

For Admin/Domain admin
Screenshot from 2020-03-27 14-48-28
For Normal user
Screenshot from 2020-03-27 14-47-58

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -1103,7 +1106,6 @@
 "sshKeyPairs": "SSH keypairs",
 "wednesday": "Wednesday",
 "noselect": "No thanks",
-"group": "Group",
```
Comment (rhtyd) [inline src/locales/en.json]: `group` is used in VM wizard, check and unremove it @shwstppr

Comment (shwstppr) [reply src/locales/en.json]: already present above in the file
https://github.com/apache/cloudstack-primate/blob/fd882b35a9a547f8a104d5ddeef34b9058ccb9a8/src/locales/en.json#L234
this was duplicate @rhtyd

**Comment by shwstppr** [comment]
@rhtyd can be reviewed

File: src/views/iam/ChangeUserPassword.vue [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+<template>
+  <div class="form-layout">
+    <a-spin :spinning="loading">
+      <a-form
+        :form="form"
+        @submit="handleSubmit"
+        layout="vertical">
+        <a-form-item v-if="!this.isAdminOrDomainAdmin()">
+          <span slot="label">
```
Comment (rhtyd) [inline src/views/iam/ChangeUserPassword.vue:26]: @shwstppr feedback for you - in forms we would like to include an information button/tooltip like this to show API parameter description/help like other auto-generated forms. There were few notification/message issues that I fixed as well.

**Pull request merged**